public/app/plugins/datasource/grafana-azure-monitor-datasource/components/ConfigEditor.tsx

```diff
@@ -102,6 +125,13 @@ export class ConfigEditor extends PureComponent<Props, State> {
             onResetOptionKey={this.resetSecureKey}
           />
         )}
+
+        {error && (
```

Comment:
how does this work with the `DataSourceSettingsPage`?
https://github.com/grafana/grafana/blob/main/public/app/features/datasources/settings/DataSourceSettingsPage.tsx#L209

Does this plugin use the above so this error ends up being rendered before the `DataSourceSettingsPage` or is this not used at all?

Reply:
gotcha 👍